src/store/concepts/movies/operations/getFavoritesMovies.js

```diff
@@ -0,0 +1,51 @@
+import { createLogic } from 'redux-logic'
+
+import apiRoutes from 'Constants/ApiRoutes'
+import { dataApiRequest, dataApiSave, dataApiSuccess } from 'Store/concepts/data/actions'
+import normalizeMovies from 'Store/schemas/movies'
+import { moviesConstant, favoritesConstant } from 'Constants/concepts'
+import { getCurrentPage } from 'Store/concepts/router/selectors'
+import storage from 'Utils/storage'
+
+import { userIdSelector } from 'Store/concepts/account/selectors'
+import { GET_FAVORITES_MOVIES } from '../types'
+import { saveFavoritesMovies } from '../actions'
+
+const getFavoritesMoviesOperation = createLogic({
+  type: GET_FAVORITES_MOVIES,
+  latest: true,
+  async process(
+    {
+      httpClient,
+      action: {
+        withoutLoading
+      },
+      getState
+    },
+    dispatch,
+    done
+  ) {
+    if (!withoutLoading) dispatch(dataApiRequest({ endpoint: favoritesConstant }))
```

Comment:
looks strange. why do you need this? if don't want to track loading state just don't show loader